### PR TITLE
Nested models

### PR DESCRIPTION
**Pull request by charan678**
# Fixes #

- Add json ingress toggle for java 
- Json ingress compatible with existing open api spec
- Add Json Model serialization/de-serialization 

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] Run `make test-docker`
- [ ] Verify affected language:
    - [ ] Generate [twilio-go](https://github.com/twilio/twilio-go) from our [OpenAPI specification](https://github.com/twilio/twilio-oai) using the [build_twilio_go.py](./examples/build_twilio_go.py) using `python examples/build_twilio_go.py path/to/twilio-oai/spec/yaml path/to/twilio-go` and inspect the diff
    - [ ] Run `make test` in `twilio-go`
    - [ ] Create a pull request in `twilio-go`
    - [ ] Provide a link below to the pull request
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-oai-generator/blob/main/CONTRIBUTING.md) and my PR follows them
- [ ] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please create a GitHub Issue in this repository.

### CODE REVIEW

File: src/test/java/com/twilio/oai/TwilioGeneratorTest.java [hunk 1]
```diff
@@ -28,14 +28,8 @@
 public class TwilioGeneratorTest {
     @Parameterized.Parameters
     public static Collection<Generator> generators() {
-        return Arrays.asList(Generator.TWILIO_CSHARP,
```
Comment (kridai) [inline src/test/java/com/twilio/oai/TwilioGeneratorTest.java:31]: I think these changes need to be reverted, guess they were part of testing.

File: src/test/java/com/twilio/oai/TwilioGeneratorTest.java [hunk 2]
```diff
@@ -47,18 +41,30 @@ public static void setUp() {
 
     @Test
     public void launchGenerator() {
-        final CodegenConfigurator configurator = new CodegenConfigurator()
-            .setGeneratorName(generator.getValue())
-            .setInputSpec("examples/spec/twilio_api_v2010.yaml")
-            .setOutputDir("codegen/" + generator.getValue())
-            .setInlineSchemaNameDefaults(Map.of("arrayItemSuffix", ""))
-            .addGlobalProperty("apiTests", "false")
-            .addGlobalProperty("apiDocs", "false");
```
Comment (kridai) [inline src/test/java/com/twilio/oai/TwilioGeneratorTest.java:56]: same comment as above here.

File: src/main/java/com/twilio/oai/resolver/ruby/RubyPropertyResolver.java [hunk 3]
```diff
@@ -2,8 +2,11 @@
 
 import com.twilio.oai.resolver.IConventionMapper;
 import com.twilio.oai.resolver.LanguagePropertyResolver;
+import org.openapitools.codegen.CodegenModel;
 import org.openapitools.codegen.CodegenProperty;
 
+import java.util.List;
```
Comment (kridai) [inline src/main/java/com/twilio/oai/resolver/ruby/RubyPropertyResolver.java:8]: nit: these imports are not used

File: src/test/java/com/twilio/oai/TwilioGeneratorTest.java (hunk 2 above)
Comment (charan678) [reply src/test/java/com/twilio/oai/TwilioGeneratorTest.java:56]: These are the changes, Changed the test script to generate code for both file and directory

File: src/test/java/com/twilio/oai/TwilioGeneratorTest.java (hunk 1 above)
Comment (charan678) [reply src/test/java/com/twilio/oai/TwilioGeneratorTest.java:31]: Will include other languages

**Review by kridai — APPROVED** [review]

**Pull request merged**